roles/openshift_openstack/defaults/main.yml

```diff
@@ -49,6 +49,12 @@ openshift_openstack_nsupdate_zone: "{{ openshift_openstack_full_dns_domain }}"
 
 
 # heat vars
+openshift_openstack_master_floating_ip: true
+openshift_openstack_infra_floating_ip: true
+openshift_openstack_app_floating_ip: true
+openshift_openstack_etcd_floating_ip: false
+openshift_openstack_load_balencer_floating_ip: true
+openshift_openstack_compute_floating_ip: false
```

Comment:
I'd say all these should default to `true` so we don't break existing interfaces.